packages/nextjs/app/my-grants/page.tsx

```diff
@@ -50,6 +51,29 @@ const MyGrants: NextPage = () => {
           )}
           <p className="m-0">{grant.description}</p>
           <p className={`badge ${badgeBgColor[grant.status]}`}>{grant.status}</p>
+          {(grant.status === PROPOSAL_STATUS.APPROVED ||
+            grant.status === PROPOSAL_STATUS.SUBMITTED ||
+            grant.status === PROPOSAL_STATUS.COMPLETED) &&
+            grant.approvedTx && (
```

Comment:
I think we could remove these checks.  If it has the approvedTx/completedTx we should just show it.